### PR TITLE
Fix distinct logins

### PR DESCRIPTION
**Pull request by mjoyce91**
Distinct logins were not populating with Oracle migration

### CODE REVIEW

File: talentmap_api/stats/views.py [hunk 1]
```diff
@@ -83,12 +83,12 @@ class UserLoginDistinctListView(mixins.ListModelMixin,
...
     Lists all logins from distinct user_ids
     '''
-    serializer_class = LoginInstanceSerializer
+    serializer_class = LoginInstanceListSerializer
     filter_class = LoginInstanceFilter
     permission_classes = (IsAuthenticated, isDjangoGroupMember('superuser'))
 
     def get_queryset(self):
-        return get_prefetched_filtered_queryset(LoginInstance, self.serializer_class).distinct('user_id').order_by('user_id')
+        return get_prefetched_filtered_queryset(LoginInstance, self.serializer_class).values('user_id').distinct()
```
Comment (elizabeth-jimenez) [inline talentmap_api/stats/views.py:91]: I think `.distinct()` is not working here? Multiples of the same `user_id` being returned.

Comment (elizabeth-jimenez) [reply talentmap_api/stats/views.py:91]: but it did fix the error that was throwing before! 🦜

Comment (mjoyce91) [reply talentmap_api/stats/views.py:91]: I'll take a look

Comment (mjoyce91) [reply talentmap_api/stats/views.py:91]: @elizabeth-jimenez which endpoint are you testing against?

Comment (mjoyce91) [reply talentmap_api/stats/views.py:91]: If you use the distinctlogins endpoint, the response does come back with a null date field, but if you track your logins through the Admin - Statistics page, the numbers come through correctly.

Comment (elizabeth-jimenez) [reply talentmap_api/stats/views.py:91]: I guess that would have been more efficient lol, I was going to the Admin Login Statistics and just printing out the return

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 🍬 👍

**Pull request merged**